File: Python/HolaMundoPro.py
import datetime as dt

def main():
    print("Ingresa tu nombre:")
    nombre = input()

    print("Ingresa tu mensaje:")
    mensaje = input()

    fecha = dt.datetime.now()

    print("Este es tu mensaje personalizado:")
    print(f"{mensaje}\n{nombre} - {fecha}")

if __name__ == "__main__":
    main()

File: Java/HolaMundoPro.java
import java.util.Scanner;
import java.util.Date;

public class HolaMundoPro {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        
        System.out.println("Ingresa tu nombre:");
        String nombre = s.nextLine();
        
        System.out.println("Escribe un mensaje:");
        String mensaje = s.nextLine();

        System.out.println("Este es tu mensaje personalizado:");

        Date fecha = new Date();

        System.out.println(mensaje + "\n" + nombre + " - " + fecha);

        s.close();
        
    }
    
}
